### PR TITLE
hwdb: quirk for kionix accelerometer on Asus TP500LB

### PR DESCRIPTION
**Pull request by M0Rf30**
This is a quirk for kionix accelerometer on Asus tp500lb. I've tested the following accel_mount_matrix setting

### CODE REVIEW

**Comment by M0Rf30** [comment]
Now it should be ok. excuse me for my git poor approach

File: hwdb/60-sensor.hwdb [hunk 1]
```diff
@@ -25,6 +25,9 @@
 # include your new rules, a description of the device, and the output of
 #   udevadm info --export-db
 #
+# For hwdb format and systemd behavior:
+#   https://www.freedesktop.org/software/systemd/man/hwdb.html#
```
Comment (AsciiWolf) [inline hwdb/60-sensor.hwdb]: Why is there a number sign at the end of url?

Comment (M0Rf30) [reply hwdb/60-sensor.hwdb]: removed

**Comment by keszybz** [comment]
Looks great, thanks!

**Comment by keszybz** [comment]
@M0Rf30 we realized that the matrix in your commit has a row full of 0s for Z direction, which means that we don't detect movement in this direction. I have posted an revert for that part (see #6431). If you could provide an updated patch, that'd be great.

**Comment by hadess** [comment]
I plan on adding a test for this in iio-sensor-proxy. See:
https://github.com/hadess/iio-sensor-proxy/issues/177

**Comment by poettering** [comment]
This entry was removed again due to #6431 being merged. @M0Rf30 any chance you can prep a new PR with the fixed entry? We can merge it again then!

**Comment by M0Rf30** [comment]
No, unfortunately I've no more physical access to this laptop, I can only say that it correctly works with this matrix, despite of missing z element in matrix

**Comment by hadess** [comment]
We'll have to wait until somebody runs into the problem again then. It's probably more likely to happen now that the iio startup race has been fixed :)

**Comment by M0Rf30** [comment]
I could investigate during the month of September. I will try to update you and do a PR if required

**Comment by hadess** [comment]
Great, thanks!

**Pull request merged**